ecosystem/sep-0024.md

```diff
@@ -741,6 +741,7 @@ Name | Type | Description
 `amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information.
 `started_at` | UTC ISO 8601 string | Start date and time of transaction.
 `completed_at` | UTC ISO 8601 string | (optional) The date and time of transaction reaching `completed` or `refunded` status. 
+`updated_at` | UTC ISO 8601 string | (optional) The date and time of transaction reaching the latest status. If the status becomes `completed` or `refunded`, this field value should be the same as `completed_at`.
```

Comment:
Why not use "current" instead of "latest"?
```suggestion
`updated_at` | UTC ISO 8601 string | (optional) The date and time of transaction reaching the current status. If the status becomes `completed` or `refunded`, this field value should be the same as `completed_at`.
```

Reply:
https://github.com/stellar/stellar-protocol/pull/1329/commits/a59e397dde665a89788a1fba1f2f588933e2c51b